src/test/java/api/loans/DueDateScheduledNoticesTests.java

```diff
@@ -82,10 +82,11 @@ public void allDueDateNoticesShouldBeScheduledOnCheckoutWhenPolicyDefinesDueDate
         beforeDueDateNoticeConfiguration,
         uponAtDueDateNoticeConfiguration,
         afterDueDateNoticeConfiguration));
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(noticePolicy)`?

Reply:
Done.